docs/Documentation/Drivers/Classic/GlobalParam.md

```diff
@@ -123,7 +123,7 @@ The following options control input and output of state files.
 | STATEYEAR             | integer   | year              | Year at which model simulation state should be saved. <br><br>*NOTE*: if STATENAME is not specified, STATEYEAR will be ignored.                                                                                                                                           |
 | STATEMONTH            | integer   | month             | Month at which model simulation state should be saved. <br><br>*NOTE*: if STATENAME is not specified, STATEMONTH will be ignored.                                                                                                                                          |
 | STATEDAY              | integer   | day               | Day at which model simulation state should be saved. State will be saved at the end of the final timestep on this day. <br><br>*NOTE*: if STATENAME is not specified, STATEDAY will be ignored.                                                                                                                                            |
-| BINARY_STATE_FILE     | string    | TRUE or FALSE     | If FALSE, VIC reads/writes the intial/output state files in ASCII format. If TRUE, VIC reads/writes intial/output state files in binary format. <br><br>*NOTE*: if INIT_STATE or STATENAME are not specified, BINARY_STATE_FILE will be ignored.                                                                                                                    |
+| STATE_FORMAT     | string    | BINARY OR ASCII        | If ASCII, VIC reads/writes the intial/output state files in ASCII format. If BINARY, VIC reads/writes intial/output state files in binary format. <br><br>*NOTE*: if INIT_STATE or STATENAME are not specified, STATE_FORMAT will be ignored.                                                                                                                    |
```

Comment:
What's the motivation for the change in case of the classic driver? Just for compatibility of the option across drivers?